Source/WebCore/crypto/cocoa/SerializedCryptoKeyWrapMac.mm

```diff
@@ -61,7 +62,11 @@
 const NSString* encryptedKeyKey = @"encryptedKey";
 const NSString* tagKey = @"tag";
 
-const size_t masterKeySizeInBytes = 16;
+constexpr size_t masterKeySizeInBytes = 16;
+constexpr size_t kekSizeInBytes = 16;
+constexpr size_t expectedTagLengthAES = 16;
+// https://datatracker.ietf.org/doc/html/rfc3394#section-2.2.1
+constexpr size_t wrappedKekSize = kekSizeInBytes + 8;
```

Comment:
Nit:  I can't tell if this comment is for the previous line or the next line.  Might be best to put it on the same line as what it's documenting.  (Not required to land the PR.)